### ISSUE TITLE
The missing variable in split_train_test.py. And ask for the text label table.

### ISSUE THREAD

**Issue by WseK** [aiotgroup/XRF55-repo#3]
First of all congrats on the paper and thanks for providing the code!

There is a variable: zoo in the code file. But you don't mention its meaning and the value of it. 

The other thing is could you send me the text label of the data? I have checked the npy files and thought you classified the data by their name. So I want to know the specific action corresponding to the label of each data.

**Comment by aiotgroup** [comment]
Thanks for your question, the variable "zoo" in split_train_test.py means that out of 20 samples of each action for each person, the first few are selected as the training set and the rest as the test set.

Our data naming format is person_action_repeats.npy, e.g. 13_22_11.npy represents the 11th repetition of the 22nd action (Jumping) for the 13th volunteer. If you want to know the name of the action that corresponds to each data, you can refer to Table 3 of our paper as follows: 

![image](https://github.com/user-attachments/assets/2b5abd98-1e95-4399-83ac-59efdbf7c1d3)

**Comment by ILGLJ** [comment]
> Thanks for your question, the variable "zoo" in split_train_test.py means that out of 20 samples of each action for each person, the first few are selected as the training set and the rest as the test set.
> 
> Our data naming format is person_action_repeats.npy, e.g. 13_22_11.npy represents the 11th repetition of the 22nd action (Jumping) for the 13th volunteer. If you want to know the name of the action that corresponds to each data, you can refer to Table 3 of our paper as follows:
> 
> ![image](https://private-user-images.githubusercontent.com/96166485/348450545-2b5abd98-1e95-4399-83ac-59efdbf7c1d3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIzMzcxODcsIm5iZiI6MTcyMjMzNjg4NywicGF0aCI6Ii85NjE2NjQ4NS8zNDg0NTA1NDUtMmI1YWJkOTgtMWU5NS00Mzk5LTgzYWMtNTllZmRiZjdjMWQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMwVDEwNTQ0N1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYwZmExYTNlZDViMzU5MWE1ODBhYzJlNGEzNTFmYzM4NGZhMjY1YTY3Njk5N2VkYmYxNGVkYzBhMjJkMjBhYTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.B6WUA-uOwEbYRvtWmTVffY1eJP8OlGx8r4lPvA0gFa8)

Thanks for your contribution! The variable "zoo" in file "split_train_test" is missing and it haven't been set before, so the file can not run
